clang/lib/DPCT/CallExprRewriterThrust.cpp

```diff
@@ -21,7 +21,30 @@ class CheckThrustArgType {
   bool operator()(const CallExpr *C) {
     std::string ArgType;
     int NumArgs = C->getNumArgs();
+
     if (Idx < NumArgs) {
+      // template<typename ELT_TYPE>
```

Comment:
I think the code example could be simplified.
// template\<typename T\>      
// void testfunc() {
// thrust::host_vector\<T\> V(1);
//  thrust::stable_sort(V.begin(),V.end(),thrust::not2(thrust::greater_equal()));
// }

Reply:
okay